src/App/Lab/Page.razor

```diff
@@ -765,6 +772,39 @@
         await ResetWorkspaceAsync(updateDiagnostics: false);
     }
 
+    private async Task FormatCurrentInputAsync()
+    {
+        if (CurrentInput is not { } input)
+        {
+            return;
+        }
+
+        if (!input.FileName.EndsWith(".cs", StringComparison.OrdinalIgnoreCase))
+        {
+            return;
+        }
+
+        try
+        {
+            var currentCode = await input.Model.GetValue(EndOfLinePreference.TextDefined, preserveBOM: true);
+
+            var syntaxTree = CSharpSyntaxTree.ParseText(currentCode);
```

Comment:
This probably won't pick up a custom compiler version if specified. I think we should move this inside the Compiler project.